package.json

```
@@ -40,7 +41,8 @@
     "parcel": "2.9.1",
     "prettier": "2.8.8",
     "process": "0.11.10",
-    "typescript": "5.2.2"
+    "typescript": "5.2.2",
+    "rimraf": "^5.0.5"
```

```suggestion
    "rimraf": "5.0.5"
```
lets lock the exact version so everyone has exact same build when developing